index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="description" content="Full-stuck developer portfolio">
+    <meta name="author" content="Darya Pogas">
+    <meta name="keywords" content="HTML, CSS, JavaScript, JS, Code The Dream">
+    <link rel="stylesheet" href="src/css/style.css"/>
```

Comment:
Small nitpick: But the instructions says to create a CSS file called` index.css` you can rename in the next assignment if you like.